### PR TITLE
Update pusher dependencies & code

### PR DESCRIPTION
**Pull request by shaqaruden**
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Proposed changes

Updates dependencies to latest available.

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red-nodes/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [x] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: social/pusher/package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+    "version": "1.0.0",
     "description": "A Node-RED node to send and receive messages using Pusher.com",
     "dependencies": {
         "pusher": "^5.1.2",
         "pusher-js": "^8.0.1"
     },
     "repository": {
         "type": "git",
-        "url": "https://github.com/node-red/node-red-nodes.git",
+        "url": "https://github.com/shaqaruden/node-red-nodes.git",
```
Comment (hardillb) [inline social/pusher/package.json:11]: This should not be changed

File: social/pusher/package.json [hunk 2]
```diff
@@ -22,8 +22,8 @@
         }
     },
     "author": {
-        "name": "Dave Conway-Jones",
-        "email": "ceejay@vnet.ibm.com",
+        "name": "Shaqaruden",
+        "email": "shaqaruden@gmail.com",
```
Comment (hardillb) [inline social/pusher/package.json:26]: This should be a `contributors` section, not replacing existing author

https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors

**Review by hardillb — CHANGES_REQUESTED** [review]

**Comment by hardillb** [comment]
Thanks for this, just a couple more points

What changed that required you to replace the icon? If pushers logo has changed please resize the new one to be match the size of the old one.

Also you will need to sign the CLA before we can accept the code. It should just be a case of clicking the link in the comment above

**Comment by shaqaruden** [comment]
> What changed that required you to replace the icon? If pushers logo has changed please resize the new one to be match the size of the old one.

Pusher has a new logo, I will resize the icon to match the old one

**Comment by shaqaruden** [comment]
> Thanks for this, just a couple more points
> 
> What changed that required you to replace the icon? If pushers logo has changed please resize the new one to be match the size of the old one.
> 
> Also you will need to sign the CLA before we can accept the code. It should just be a case of clicking the link in the comment above

All completed.

**Review by hardillb — APPROVED** [review]

**Comment by hardillb** [comment]
@dceejay this looks good to me, please merge and push to npm when you can (might want to decide if we bump the version to 0.2.0/1.0.0 rather than 0.1.1)

**Pull request merged**